Code/Framework/AzFramework/AzFramework/Physics/ShapeConfiguration.h

```diff
@@ -8,14 +8,21 @@
 
 #pragma once
 
-#include <AzCore/Math/Vector3.h>
-#include <AzCore/Math/Vector2.h>
-#include <AzCore/Math/Quaternion.h>
 #include <AzCore/Asset/AssetCommon.h>
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/Math/Transform.h>
+#include <AzCore/Math/Vector2.h>
+#include <AzCore/Math/Vector3.h>
 #include <AzCore/Serialization/SerializeContext.h>
-
 #include <AzFramework/Physics/HeightfieldProviderBus.h>
 
+namespace AZ
+{
+    class Capsule;
+    class Obb;
+    class Sphere;
+} // namespace AZ
```

Comment:
little confused how you can forward declare these when you are returning a concrete type below?